### DISCUSSION TITLE
SoC not updated

### DISCUSSION THREAD

**Discussion by toemk1** [tillsteinbach/WeConnect-mqtt#305] [Q&A]
Hi and thanks for weconnect! I'm using the package with grafana as a docker image and thats working great (including current SoC). For the integration in FHEM i use the cli-mqtt-Version (0.48.3) as well. The latter one doesnt show any errors but is reporting 80% SoC for weeks now. Any ideas why?

T.

**Comment by toemk1** [comment]
correction: currentSOC_pct always jumps back to 80% when there has been no update by the car since the last interval. cruisingRangeElectric_km stays "correct" by the way....

**Reply by tillsteinbach** [reply]
Can you check with any other mqtt browser that it is not caused by FHEM?

**Reply by toemk1** [reply]
I'll have to set that up first. Is there any difference how the grafana-package handels that? Because there its always the correct SoC. Or can I pull the value from there instead of running a second process? Thanks!

**Comment by tillsteinbach** [comment]
If you mean VWsFriend, yes it stores the value in the database while mqtt is just the live data coming from the servers
